ui/components/app/tab-bar/index.scss

```diff
@@ -53,6 +56,17 @@
           min-height: 14px;
         }
       }
+
+
+      &__icon {
+        margin-right: 16px;
```

Comment:
For anything `-right` or `-left`, we should use `-inline-right` or `inline-left` to better support RTL.

Reply:
There are no -inline-right and -inline-left options. Will use -inline-end for this one.